Formula/xkbcomp.rb

```diff
@@ -20,9 +21,17 @@ class Xkbcomp < Formula
   depends_on "libxkbfile"
 
   def install
-    system "./configure", *std_configure_args
+    # avoid cellar in xkb data dir
+    destdir = buildpath/"dest"
+    system "./configure", "--prefix=#{HOMEBREW_PREFIX}"
     system "make"
-    system "make", "install"
+    system "make", "DESTDIR=#{destdir}", "install"
+    prefix.install Dir["#{destdir}#{HOMEBREW_PREFIX}/*"]
+    inreplace lib/"pkgconfig/xkbcomp.pc" do |s|
+      s.gsub! HOMEBREW_PREFIX, opt_prefix
+      # let `datarootdir`` = HOMEBREW_PREFIX/share
+      s.gsub! %r{\${prefix}(?=/share)}, HOMEBREW_PREFIX
```

Comment:
Don't you need to escape the `{` and `}` too? Also, maybe `change_make_var!` might be cleaner for this line.